lib/mkmf-rice.rb

```diff
@@ -1,7 +1,7 @@
 require 'mkmf'
 
-IS_MSWIN = RbConfig::CONFIG['host_os'].match?(/mswin/)
-IS_MINGW = RbConfig::CONFIG['host_os'].match?(/mingw/)
+IS_MSWIN = /mswin/ =~ RUBY_PLATFORM
```

Comment:
Oh I meant actually using the $constants defined my mkmf already. Unless their and not accessible from the extconf.rb file?

Reply:
Ah, seems like they should be accessible. Feel free to update in a follow-up commit.